gradle.properties

```diff
@@ -7,26 +7,22 @@ org.gradle.jvmargs         = -Xmx3G
 org.gradle.daemon          = false
 org.gradle.debug           = false
 
-# Project
-group                      = mod.azure.azurelib
-version                    = 2.1.6
```

Comment:
Removing this breaks the maven uploading if I remember correctly now, will have to test